src/Okta.Idx.Sdk/AuthenticationStatus.cs

```diff
@@ -0,0 +1,27 @@
+﻿// <copyright file="AuthenticationStatus.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Okta.Idx.Sdk
+{
+    /// <summary>
+    /// This enum represents all the possible authentication response statuses
+    /// </summary>
+    public enum AuthenticationStatus
```

Comment:
Since these are technically ENUMS, and in i believe all cases with Okta, enums are `caps`... should these be `SUCCESS` and `PASSWORD_EXPIRED`?

Reply:
We actually use [PascalCase for enums in .NET](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations).